libs/core/src/lib/fixed-card-layout/fixed-card-layout.component.ts

```diff
@@ -88,9 +109,14 @@ export class FixedCardLayoutComponent implements OnInit, AfterContentInit, After
     /** @hidden */
     private _previousNumberOfColumns: number;
 
+    private _cardsArray: Array<CardDefinitionDirective>;
+
     /** An RxJS Subject that will kill the data stream upon component’s destruction (for unsubscribing)  */
     private readonly _onDestroy$: Subject<void> = new Subject<void>();
 
+    // FocusKeyManager instance
+    private keyboardEventsManager: FocusKeyManager<FixedCardLayoutItemComponent>;
```

Comment:
should be prefixed with `_` since it's private

Reply:
fixed this.
and other places where wrong accessor was there.